app/models/labware_creators/pooled_tubes_from_whole_tubes.rb

```diff
@@ -39,33 +38,28 @@ def barcodes=(input)
 
     # TODO: This should probably be asynchronous
     def available_tubes
```

Comment:
Does this method find tubes that are appropriate to be the source tubes? (correct purpose)

Reply:
Yep, it tries to exclude those that have been used, but there's another bug elsewhere in Limber that means it isn't always 100% successful in this. (I've added a story here as it seemed to be missing https://github.com/sanger/limber/issues/645)

I also noticed that there seem to be a few tubes 'abandoned' at this stage. Some of them were completed on another fork, others looks just genuinely abandoned. The former we could probably add a scope to filter out, but the latter will always hand about.

There's a limit to confine the returned tubes to the 30 most recent though, so this list shouldn't get too unmanageable.